### DISCUSSION TITLE
第5节课笔记(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#458] [笔记]
20班的笔记在这里提交哈

**Comment by liu673** [comment]
https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_5/lesson_5_note.md

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/679625950

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/679659661

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135873703
